These `git pull` invocations will operate on the parent repository, meaning
`git clone` won't be called reliably.

diff --git a/build/Makefile.gobin b/build/Makefile.gobin
index e3e37d8..b906e8a 100644
--- a/build/Makefile.gobin
+++ b/build/Makefile.gobin
@@ -31,14 +31,14 @@ fetch-go-tpm-tools:
 	$(Q)echo "Making proto dependencies dir..."
 	$(Q)mkdir -p proto_deps
 	$(Q)echo "Cloning/updating go-tpm-tools..."
-	$(Q)git -C proto_deps/go-tpm-tools pull 2> /dev/null || \
+	$(Q)test -d proto_deps/go-tpm-tools/.git || \
 		git clone $(GO_TPM_TOOLS_URL) proto_deps/go-tpm-tools
 
 fetch-mwlid-proto:
 	$(Q)echo "Making proto dependencies dir for mwlid..."
 	$(Q)mkdir -p proto_deps/mwlid
 	$(Q)echo "Cloning/updating mwlid-proto..."
-	$(Q)git -C proto_deps/mwlid pull 2> /dev/null || \
+	$(Q)test -d proto_deps/mwlid/.git || \
 		git clone $(MWLID_PROTO_URL) proto_deps/mwlid
 	$(Q)mkdir -p cmd/core_plugin/workloadcertrefresh/proto
 	$(Q)cp proto_deps/mwlid/proto/v1/workload-identity.proto cmd/core_plugin/workloadcertrefresh/proto/workload_identity.proto
